spec/index.bs

```diff
@@ -1304,6 +1304,7 @@ typedef (GPUOutOfMemoryError or GPUValidationError) GPUError;
 <script type=idl>
 partial interface GPUDevice {
     void pushErrorScope(GPUErrorFilter filter);
+    // Rejects with DOMException("OperationError") if there are no error scopes on the stack.
```

Comment:
Update and move this text out of the IDL block

Reply:
Done.